R/fit_model.R

```diff
@@ -85,6 +85,19 @@ fit_model <- function(
   )
 }
 
+#' Format Stan options for input to EpiNow2
+#' 
+#' Format configuration `sampler_opts` for input to EpiNow2 via a call to
+#' [EpiNow2::stan_opts()].
+#'
+#' @param sampler_opts A list containing the entries `cores`, `chains`,
+#' `iter_warmup`, `iter_sampling`, `adapt_delta` and `max_treedepth`. If any of
+#' these entries are missing an error will be returned
+#' @param seed A stochastic seed passed here to the Stan sampler and as the R
+#' PRNG seed for EpiNow2 initialization
+#' 
```

Comment:
Can you add a @ return arg?

Reply:
Yes

Reply:
[8881d6f](https://github.com/CDCgov/cfa-epinow2-pipeline/pull/139/commits/8881d6f5017a9221e49f886a64b7ebaba55b28bd)